src/main/java/frc/robot/controls/Operator.java

```diff
@@ -1,37 +1,134 @@
 package frc.robot.controls;
 
+import java.util.function.BooleanSupplier;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.PrintCommand;
+import frc.robot.commands.arm.ExtendToPosition;
+import frc.robot.commands.DoNothing;
+import frc.robot.constants.ArmConstants;
 import frc.robot.commands.deployingbar.RotateDeployingBar;
 import frc.robot.constants.DeployingBarConstants;
 import frc.robot.constants.OIConstants;
+import frc.robot.subsystems.Drivetrain;
 import frc.robot.subsystems.FourBarArm;
 import frc.robot.subsystems.DeployingBar;
 import frc.robot.subsystems.Intake;
+import frc.robot.util.Node;
+import frc.robot.util.Vision;
 import lib.controllers.GameController;
+import lib.controllers.GameController.Axis;
 import lib.controllers.GameController.Button;
+import lib.controllers.GameController.DPad;
 
 public class Operator {
 
-  private GameController operator = new GameController(OIConstants.kOperatorJoy);
+  private GameController m_operator = new GameController(OIConstants.kOperatorJoy);
+
+  private Drivetrain m_drive;
+  private FourBarArm m_arm;
+  private Vision m_vision;
+  private Intake m_intake;
+  private DeployingBar m_deployingBar;
+
+  // Values for selecting a node
+  // Grid, row, and column in the grid
+  private int[] selectValues = {1, 1, 1};
+
+  // The selected node where the robot will score
+  public static Node selectedNode = new Node();
```

Comment:
generaly don't make viarables public or static.

Reply:
I made it private, but I don't want to pass the operator into the driver, which I would have to do if it wasn't static.